src/pybind/mgr/dashboard/tox.ini

```diff
@@ -20,6 +20,7 @@ addopts =
 deps =
     -rrequirements.txt
     -cconstraints.txt
+    PyJWT
```

Comment:
comparing to the original PR, I don't see this change there in the last commits, https://github.com/ceph/ceph/pull/54710/files. Was this intentional?

Reply:
@mcv21 probably the order of the commits might be a reason for it. I see that you cherry-picked them in the reverse order.

Reply:
Oh, did I? Bother, let me check

Reply:
Sorry, my bad, let me re-do this branch with the cherry-picks in the right order :facepalm:

Reply:
@nizamial09 sorry about that! I've double-checked the cherry-pick order, and updated the MR text to note that all three applied cleanly.